resources/ebsco_kb.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+/*
+**************************************************************************************************************************
+** CORAL Resources Module v. 1.2
+**
+** Copyright (c) 2010 University of Notre Dame
+**
+** This file is part of CORAL.
+**
+** CORAL is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+**
+** CORAL is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License along with CORAL.  If not, see <http://www.gnu.org/licenses/>.
+**
+**************************************************************************************************************************
+*/
+
+
+
+include_once 'directory.php';
+
+
+//used for creating a "sticky form" for back buttons
+//except we don't want it to retain if they press the 'index' button
+//check what referring script is
+
+CoralSession::set('ref_script', $currentPage);
```

Comment:
I got the message
> PHP Notice:  Undefined variable: currentPage

when loading the ebsco_kb.php page.

Reply:
fixed